packages/lumx-react/src/components/dialog/Dialog.tsx

```diff
@@ -121,6 +126,15 @@ export const Dialog: Comp<DialogProps, HTMLDivElement> = forwardRef((props, ref)
           }
         : undefined;
 
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    const [sentinelTop, setSentinelTop] = useState(null);
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    const [sentinelBottom, setSentinelBottom] = useState(null);
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    const intersections = useIntersectionObserver([sentinelTop, sentinelBottom], {
+        threshold: [0, 1],
+    });
+
```

Comment:
I didn't mean to move this. 
I just removed it and added it back in :sweat_smile: I'll move it back down